### ISSUE TITLE
Error: Node type 'ClassPrivateMethod' is not supported yet

### ISSUE THREAD

**Issue by meezwhite** [putoutjs/minify#3]
When using `minify index.js > index.min.js` and `index.js` includes a class with a `#privateMethod`, error stack:

```bash
Error: Node type 'ClassPrivateMethod' is not supported yet: '#privateMethod(args) { return; }'
    at maybe.maybeThrow (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1489432)
    at f (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1493018)
    at file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1493488
    at classVisitor (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1442303)
    at _class.ClassDeclaration (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1442614)
    at maybe.maybePlugin (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1489817)
    at f (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1493133)
    at file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1493488
    at Array.forEach (<anonymous>)
    at Program (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1478485)
```

Similar to #2

**Comment by coderaiser** [comment]
Fixed in [`@putout/printer@1.105.0`](https://github.com/putoutjs/printer/releases/tag/v1.105.0) 🎉 . Landed in [`@putout/minify@1.17.0`](https://github.com/putoutjs/minify/releases/tag/v1.17.0).

Is it works for you?

**Comment by meezwhite** [comment]
Great thanks! `ClassPrivateMethod` works now!
